priv/repo/migrations/20240329123743_generate_dataclip_search_vectors.exs

```diff
@@ -0,0 +1,42 @@
+defmodule Lightning.Repo.Migrations.GenerateDataclipSearchVectors do
+  require Logger
+  use Ecto.Migration
+
+  def change do
+    execute(
+      # Calculate the search vectors in batches of 1000 rows until all rows have
+      # been processed.
+      fn ->
+        Stream.unfold(1, fn n ->
+          if n > 0 do
+            %{num_rows: n} = update_search_vector(1000)
+            Logger.info("Updated search_vector for #{n} rows")
+
+            {:ok, n}
+          else
+            nil
+          end
+        end)
+        |> Stream.run()
+      end,
+      ""
+    )
+  end
+
+  defp update_search_vector(limit) do
+    repo().query!(
+      """
+      UPDATE dataclips
+      SET search_vector = jsonb_to_tsvector('english_nostop', body, '"all"')
```

Comment:
the `'"all"'` is magic... it tells postgres to include the keys and the values in the index